src/components/navbar/NavBar.tsx

```diff
@@ -64,16 +66,26 @@ export default function NavBar() {
     </Toolbar>
   );
 
-  const NavItem = ({ href, icon, text, onClick }: NavItemProps) => (
-    <ListItem disablePadding className={$navbar.linkItem}>
-      <Link href={href || ""} className={$navbar.link}>
-        {icon}
-        <p className={$navbar.linkTitle} onClick={onClick}>
-          {text}
-        </p>
+  const NavItem = ({ href, icon, text, onClick }: NavItemProps) => {
+    const currentPath = router.pathname;
+    const selectedPath = currentPath.includes(text.toLowerCase());
```

Comment:
🐧 probably would be a bit more robust to compare on `href` instead of the display name

Reply:
![Screenshot 2023-11-16 at 9 31 26 AM](https://github.com/sfbrigade/compass/assets/104481165/0bfee408-1323-438c-bd01-d692b9f32aeb)